content/highlights/diversity_of_life.md

```diff
@@ -0,0 +1,39 @@
+---
+title: New study shows crossing the salt barrier has shaped the diversity of life
+date: 2022-10-19
+summary: TStudy by Jamy *et al.* show that hundreds of transitions have occurred between marine and non-marine habitats over the course of two billion years of eukaryotic evolution. Code and data are shared openly.
+banner: /highlights/diversity_of_life_thumbnail.png
+banner_large: /highlights/diversity_of_life.png
+banner_caption: "Fig 1 from Jamy et al. (2022), Global eukaryotic 18S–28S phylogeny from environmental samples and the distribution of habitats."
+tags: [Highlight, Bayesian, Paleovegetation, Deep-learning]
```

Comment:
these are not the right tags. We need to think of some. Ask Katarina. Highlight has to stay, try link to any related highlights with similar key words